host/xtest/xtest_main.c

```diff
@@ -150,6 +154,12 @@ int main(int argc, char *argv[])
 
 	opterr = 0;
 
+	if (argc > 1 && !strcmp(argv[1], "sha-perf")){
```

Comment:
The busybox way is an interesting hack to save memory by giving different personalities to the same binary depending on with which name it's started. As we don't really need it I'd rather avoid it to keep things simple.